### DISCUSSION TITLE
PyPi package

### DISCUSSION THREAD

**Discussion by dimk00z** [factory5group/django-liveconfigs#1] [Ideas]
Добрый день, после митапа у Evrone хотел попробовать в действии, но пакета нет в PyPI=((
Есть в планах добавить?

**Comment by DenisDudnik** [comment]
Добрый день. Планы есть конечно. Некоторые время назад PyPI закрывал возможность регистрации и размещения новых пакетов из-за борьбы с вирусами. Если это ограничение убрали, то следующую версию разместим уже там.
На текущий момент можно установить пакет отсюда. Пример такой установки можно найти в репозитории, который указан в самом низу файла readme.
Если лень искать, то просто добавь строку в свой requirements.txt
```
django-liveconfigs @ git+https://github.com/factory5group/django-liveconfigs.git@v1.0.2
```
Ссылка на сам файл https://github.com/factory5group/django-liveconfigs-example/blob/main/requirements/requirements.txt

**Reply by dimk00z** [reply]
Спасибо за ответ,  я видел в примере ссылку, но это не совсем продакш подход. Поэтому и написал

**Reply by DenisDudnik** [reply]
Согласен. Pypi обязательно будет.

**Comment by DenisDudnik** [comment]
> Добрый день, после митапа у Evrone хотел попробовать в действии, но пакета нет в PyPI=(( Есть в планах добавить?

Пакет уже есть в pypi

**Reply by dimk00z** [reply]
Спасибо
